libs/vm_monitor/src/cgroup.rs

```diff
@@ -85,7 +88,10 @@ impl CgroupWatcher {
 
         // buffer for samples that will be logged. once full, it remains so.
         let history_log_len = self.config.memory_history_log_interval;
+        let max_skip = self.config.memory_history_log_noskip_interval;
         let mut history_log_buf = vec![MemoryStatus::zeroed(); history_log_len];
+        let mut last_logged_memusage = MemoryStatus::zeroed();
+        let mut can_skip_logs_until = Instant::now() - max_skip;
```

Comment:
Why `now() - max_skip` ? (maybe I'm missing something, but won't `now()` suffice, because inside the loop `now()` will be newer than it?)

Reply:
Ah, I see - because `Instant`s are only guaranteed to be non-decreasing?

Let's add a brief comment saying that?